src/components/Clock/Clock.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export class Clock extends React.Component {
+  state = {
+    time: new Date(),
+    intervalID: null,
+  }
+
+  componentDidMount() {
+    this.setState({
+      intervalID: setInterval(() => {
+        this.setState({ time: new Date() });
+        // eslint-disable-next-line
+        console.log(this.state.time.toLocaleTimeString())
+      }, 1000),
```

Comment:
```suggestion
  componentDidMount() {
    this.intervalID = setInterval(() => {
        this.setState({ time: new Date() });
        // eslint-disable-next-line
        console.log(this.state.time.toLocaleTimeString())
      }, 1000),
```
No need to write setInterval into the state